nvtabular/column_selector.py

```diff
@@ -84,3 +91,9 @@ def __radd__(self, other):
 
     def __rshift__(self, other):
         return nvtabular.ColumnGroup(self) >> other
+
+    def __eq__(self, other):
+        if isinstance(other, ColumnSelector):
```

Comment:
The implementation looks right here. It might be clearer to first check if `other` is _not_ a `ColumnSelector` and immediately return `False` if so, because then the `if` statements wouldn't need to be nested and the return values would be close together with the conditions that return them.